LayoutTests/imported/w3c/web-platform-tests/css/css-animations/display-none-to-display-block-dont-cancel.tentative.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<link rel="author" title="Tim Nguyen" href="https://github.com/nt1m">
+<link rel="help" href="https://drafts.csswg.org/css-display-4/#display-animation">
+<link rel="help" href="https://github.com/w3c/csswg-drafts/issues/6429">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/css-animations/support/testcommon.js"></script>
+
+<div id="target">hello</div>
+<style>
+@keyframes display-animation {
+  0% { display: none; }
+  100% { display: block; }
+}
+#target {
+  display: none;
+}
+#target.animate {
+  animation: display-animation 1s;
+  display: block;
+}
+</style>
+<script>
+promise_test(async (t) => {
+  t.add_cleanup(() => target.classList.remove('animate'));
```

Comment:
Do we need this?

Reply:
Same, it's name to reset the state of the test so they can be re-ran if needed when opened manually